src/session/session_api.c

```diff
@@ -52,29 +52,58 @@ __wt_session_reset_cursors(WT_SESSION_IMPL *session, bool free_buffers)
  *     Sweep the cursor cache.
  */
 int
-__wt_session_cursor_cache_sweep(WT_SESSION_IMPL *session)
+__wt_session_cursor_cache_sweep(WT_SESSION_IMPL *session, bool big_sweep)
 {
+    WT_CONNECTION_IMPL *conn;
     WT_CURSOR *cursor, *cursor_tmp;
     WT_CURSOR_LIST *cached_list;
     WT_DECL_RET;
 #ifdef HAVE_DIAGNOSTIC
     WT_DATA_HANDLE *saved_dhandle;
 #endif
-    uint64_t now;
-    uint32_t position;
-    int i, t_ret, nbuckets, nexamined, nclosed;
+    uint64_t now, sweep_max, sweep_min;
+    uint32_t i, nclosed, position;
+    int t_ret, nbuckets, nexamined;
     bool productive;
 
     if (!F_ISSET(session, WT_SESSION_CACHE_CURSORS))
         return (0);
 
+    conn = S2C(session);
+
     /*
      * Periodically sweep for dead cursors; if we've swept recently, don't do it again.
+     *
+     * Each call of this sweep function visits all the cursors in some number of buckets used by the
+     * cursor cache. This determines if any of the visited cursors reference dead or dying data
+     * handles and should be fully closed and evicted from the cache. Evicting a cursor from the
```

Comment:
If possible I'd suggest finding different words instead of `evicted from the cache` because that could be confusing for someone thinking you're referring to the data cache and eviction threads, etc.

Reply:
Fixed, now: `removed from the cache`